components/kyma-environment-broker/internal/dto.go

```diff
@@ -224,26 +224,26 @@ type ERSContext struct {
 	Region                *string                            `json:"region,omitempty"`
 }
 
-func UpdateERSContext(provisioning, lastOperation ERSContext) ERSContext {
-	if lastOperation.SMOperatorCredentials != nil {
-		provisioning.SMOperatorCredentials = lastOperation.SMOperatorCredentials
+func UpdateERSContext(op, lastOperation ERSContext) ERSContext {
```

Comment:
small refactor, `UpdateERSContext` is used on other types of operations than jus the provisioning. It should ensure that `ERSContext` is "inherited" by the following operations if not present in the payload from ERS